### PR TITLE
User Model Tests

### PR DESCRIPTION
**Pull request by dborski**
## Description
Completes User Story #48 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Adds specs for
- Get user by ID
- Get all users
- Update user style preference
- Deletes a user

## RSpec Results
- All tests passing
- User login test skipped for now

### CODE REVIEW

File: code/api/src/modules/user/query.test.js [hunk 1]
```diff
@@ -17,7 +18,43 @@ describe('user queries', () => {
         schema: schema,
         graphiql: false,
       }),
-    )
+    );
+    await models.User.destroy({ where: {} })
+  })
```
Comment (jhgould) [inline code/api/src/modules/user/query.test.js:23]: Very cool. Glad you figure out how to do this.

**Review by jhgould — APPROVED** [review]

File: code/api/src/modules/user/query.test.js [hunk 2]
```diff
@@ -17,7 +18,43 @@ describe('user queries', () => {
...
+
+    await models.User.create(userData1);
+    await models.User.create(userData2);
+  })
+
+  afterEach(async () => {
+    await models.User.destroy({ where: {} })
+  })
+
+  afterAll(() => {
```
Comment (dborski) [inline code/api/src/modules/user/query.test.js:56]: These functions will ensure that our tests are torn down properly in order to stop tests from "bleeding" into each other

**Review by muydanny — APPROVED** [review]

File: code/api/src/modules/user/query.test.js (hunk 2 above)
Comment (muydanny) [reply code/api/src/modules/user/query.test.js:56]: This is awesome!

**Pull request merged**